keras_cv/models/models_test.py

```diff
@@ -69,7 +99,12 @@ def test_application_base(self, app, _, args):
 
         # Can be serialized and deserialized
         config = model.get_config()
-        reconstructed_model = model.__class__.from_config(config)
+        if "ConvNeXt" in app.__name__:
+            custom_objects = {"LayerScale": convnext.LayerScale}
```

Comment:
will registering this with @tf.keras.utils.register_keras_serializable make us not need this?

Reply:
Could you provide an example?

Reply:
https://github.com/keras-team/keras-cv/blob/master/keras_cv/layers/preprocessing/aug_mix.py#L24

Basically loading our models should not require custom object registration - so lets give this a shot.